src/main/java/step2/lotto/domain/WinningCondition.java

```diff
@@ -0,0 +1,31 @@
+package step2.lotto.domain;
+
+public enum WinningCondition {
+	NOTHING(0, 0),
+	FOURTH_PRIZE(3, 5000),
```

Comment:
```suggestion
	FOURTH_PRIZE(3, 5_000),
```
가독성을 위해 추가해 주세요, 실수하기 쉽습니다.